src/Data/TypeRep/Map.hs

```diff
@@ -18,7 +19,6 @@ module Data.TypeRep.Map
          TypeRepMap (..)
 
          -- * Construction
-       , empty
```

Comment:
I am not sure about this. As far as I know, all map-like structures have their own `empty` 🤔 Probably sometimes it could be better to use it instead of something like `newIORef @TMap mempty`?

Reply:
@vrom911 That's a good point. I will return `empty` back.